### DISCUSSION TITLE
Embedding chatbot in my website

### DISCUSSION THREAD

**Discussion by hibaq86** [FlowiseAI/Flowise#1132] [Q&A]
I have set up flowise on my laptop and have linked it to my Azure account. I have created a chatbot using my vectara account and even though the chatbot works in the flowise back end, every time I try and use the embedding code in the body of my website, it is stuck on thinking. I am trying to upload multiple files to my chatbot, does anybody have any suggestions?

**Comment by blissiraDC** [comment]
I had the same problem with my deployment on AWS. I suspect it might be the same issue on Azure. My problem was due to there being no SSL certificate coming from AWS and my Hosting on Hostinger was blocking it. I had to set up Flowise to run through a domain name I had set up by configuring NGNIX. Go to your website, right-click and choose Inspect. If this is the same problem you will see Error messages  messages stating an SSL issue.

**Reply by hibaq86** [reply]
Why does the chatbot work when I do the quick start, but not when I clone the repository? Am I supposed to clone the regular flowise repository or just the embed folder? Also, yarn dev won't work for me

**Reply by vskavo** [reply]
I'm having the same issue on Azure, how did you manage to set up Flowise with a domain name and NGNIX? I've tryed a self-signed certificate, but didn't work. Thanks in advance!

**Reply by blissiraDC** [reply]
I followed the flowise docs for AWS setup the last step is to connect to a domain (Install Certbot to have HTTPS) . I don't know I have never used Azure but it may be possible its a similar process for Azure or at least an option to have HTTPS somewhere?
